channels/rdpgfx/server/rdpgfx_main.c

```diff
@@ -345,8 +345,8 @@ static UINT rdpgfx_send_create_surface_pdu(RdpgfxServerContext* context,
  *
  * @return 0 on success, otherwise a Win32 error code
  */
-UINT rdpgfx_send_delete_surface_pdu(RdpgfxServerContext* context,
-                                    RDPGFX_DELETE_SURFACE_PDU* pdu)
+static UINT rdpgfx_send_delete_surface_pdu(RdpgfxServerContext* context,
+        RDPGFX_DELETE_SURFACE_PDU* pdu)
```

Comment:
I noticed this format change. Why it looks different rule from rdpgfx_send_solid_fill_pdu below?